src/storage/hummock_sdk/src/compaction_group/hummock_version_ext.rs

```diff
@@ -894,6 +927,7 @@ pub fn build_initial_compaction_group_levels(
             vnode_partition_count: 0,
         });
     }
+    #[expect(deprecated)] // for backward-compatibility of previous hummock version delta
```

Comment:
Are we planning to deprecate the whole Levels struct? I thought only `member_table_ids` will be deprecated.

Reply:
Yes, only the field is deprecated. The annotation is applied over the whole struct because we can't apply on a single fie.d